src/utils/elementToSvg.ts

```diff
@@ -0,0 +1,35 @@
+import type { SatoriOptions } from "satori";
+import satori from "satori/wasm";
+
+import { loadGoogleFont } from "./loadGoogleFont.js";
+import { type HonoElement, toReactNode } from "./toReactNode.js";
+
+export type ElementToSvgOptions = Partial<SatoriOptions> & {
+  width?: number;
+  height?: number;
+};
+
+export async function elementToSvg(
+  element: HonoElement,
+  options: ElementToSvgOptions = {},
+) {
+  const { fonts, height = 630, width = 1200 } = options;
+
+  const svg = await satori(toReactNode(element), {
+    ...options,
+    width,
+    height,
```

Comment:
This is needed for emojis and multi-language support: https://github.com/vercel/satori/blob/main/playground/pages/index.tsx#L561

Reply:
ty

Reply:
done https://github.com/wevm/hono-og/pull/28/commits/c4abb1c05ad8acbec214a572d6ac652973eaba19